plugins/woocommerce/src/Internal/Features/ProductBlockEditor/ProductTemplates/SimpleProductTemplate.php

```diff
@@ -1186,8 +1186,8 @@ private function add_linked_products_group_blocks() {
 				'attributes'     => array(
 					'title'       => __( 'Cross-sells', 'woocommerce' ),
 					'description' => sprintf(
-						/* translators: %1$s: Learn more about linked products. %2$s: Learn more about linked products.*/
-						__( 'By suggesting complementary products in the cart using cross-sells, you can significantly increase the average order value. %1$sLearn more about linked products.%2$s', 'woocommerce' ),
+						/* translators: %1$s: Learn more about linked products. %2$s: Learn more about linked products link closing tag.*/
```

Comment:
I like the clarification here... should update `%1$s` as well!

Reply:
Done in the commit [1983981](https://github.com/woocommerce/woocommerce/pull/46346/commits/1983981e9ff6bb1d7440f72a89a918f65e52ad46)